src/main/java/com/savvato/tribeapp/repositories/NounRepository.java

```diff
@@ -1,13 +1,16 @@
 package com.savvato.tribeapp.repositories;
 
-import com.savvato.tribeapp.entities.Adverb;
-import com.savvato.tribeapp.entities.Verb;
+import com.savvato.tribeapp.entities.Noun;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
 import java.util.Optional;
 
 @Repository
-public interface NounRepository extends CrudRepository<Adverb, Long> {
-    Optional<Verb> findByWord(String word);
+public interface NounRepository extends CrudRepository<Noun, Long> {
+    Optional<Noun> findByWord(String word);
+
+    @Query(nativeQuery = true, value = "select word from noun where id = ?")
+    Optional<String> findNounById(Long id);
```

Comment:
should be findById()

Reply:
Will fix according to what we decide in Adverb conversation above.